src/components/ebay-carousel/utils/scroll-transition/index.js

```diff
@@ -21,7 +21,7 @@ module.exports = function scrollTransition(el, to, fn) {
                 return fn();
             }
 
-            el.scrollLeft = easeOut(delta / duration) * distance + scrollLeft;
```

Comment:
This is also not a necessary change however easeOut was chosen with the intent that the carousel would still be moving when this animation happened. Since we are currently waiting for the scrolling to stop before animating `ease-in-out` looks much better. Again this is a very low risk change.